internal/bminventory/inventory.go

```diff
@@ -1502,6 +1502,24 @@ func (b *bareMetalInventory) refreshClusterHosts(ctx context.Context, cluster *c
 	return nil
 }
 
+func (b *bareMetalInventory) noneHaModeClusterUpdateValidations(cluster *common.Cluster, params installer.UpdateClusterParams) error {
+
+	if swag.StringValue(cluster.HighAvailabilityMode) != models.ClusterHighAvailabilityModeNone {
+		return nil
+	}
+	msgPrefix := "is not allowed in single node mode"
```

Comment:
Prefix or suffix? :)
But I really think you should just repeat it in every message instead of it